.github/workflows/create_release.yml

```diff
@@ -92,3 +94,13 @@ jobs:
           verbose: true     
           print-hash: true  
 
+
+  test_src_dist:    
```

Comment:
nit: please remove trailing spaces here and everywhere else. There should be an editor settings to automate that. Sorry for being nitpicky on this